### ISSUE TITLE
Tabs lack ARIA markup that describes the purpose and interaction of the tab elements 

### ISSUE THREAD

**Issue by mcgratay** [pressbooks/pressbooks#1706]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

Add `role=”tablist”` to the tab `<ul>` and `role=”tab”` to individual `<li>` elements. On the tab content divs, add `role=”tabpanel”` and `aria-labeledby=id` of its tab. Set `aria-selected=”true”` for selected tabs, and `aria-hidden=”true”` for hidden tab panels. For an example, see:  http://accessibility.athena-ict.com/aria/examples/tabpanel2.shtml

Tabs
- Visual and text editors: chapter, front matter, back matter, glossary terms, copyright notice, long description, cover generator about section
- Theme Options

### Steps to Reproduce

1. Access an existing book on Pressbooks and navigate to a chapter
2. Use assistive technology to navigate between the text and visual editors 

**Expected behavior:** The role of the tab is announced using AT 

**Actual behavior:** ARIA markup for roles is not included in tabs

**Comment by connerbw** [comment]
Maybe?

https://make.wordpress.org/core/2019/04/02/admin-tabs-semantic-improvements-in-5-2/
